lesson001/src/main/java/lesson5/Personal.java

```diff
@@ -0,0 +1,38 @@
+package lesson5;
+
+public class Personal {
+    private String name;
+    private String pos;
+    private String email;
+    private String phone;
+    private int age;
+    private int salary;
+
+    public Personal(String name, String pos, String email, String phone, int age, int salary){
+        this.name = name;
+        this.pos = pos;
+        this.email = email;
+        this.phone = phone;
+        this.age = age;
+        this.salary = salary;
+
+    }
+
+    public Personal(String name, String pos, int age){
+        this(name, pos, "unknown","unknown", age, 100000 );
+
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void PersonalInfo(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.